cdap-data-fabric/pom.xml

```diff
@@ -45,13 +45,6 @@
     </dependency>
     <dependency>
       <groupId>co.cask.cdap</groupId>
-      <artifactId>cdap-common</artifactId>
-      <version>${project.version}</version>
-      <type>test-jar</type>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>co.cask.cdap</groupId>
```

Comment:
This is duplication of lines 39 to 45. If I leave this change in, `maven` complains:

```
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for co.cask.cdap:cdap-data-fabric:jar:3.5.5
[WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: co.cask.cdap:cdap-common:test-jar -> duplicate declaration of version ${project.version} @ co.cask.cdap:cdap-data-fabric:[unknown-version], /Users/john/Source/cdap_3.5/cdap-data-fabric/pom.xml, line 46, column 17
[WARNING] 

```
and then the build eventually hangs.